src/Handlers/historyHandler.js

```diff
@@ -0,0 +1,47 @@
+import { Abbreviations } from "../services/abbreviations";
+import {
+  clearHistoryService,
+  deleteHistoryService,
+  postHistoryService,
+} from "../services/historyService";
+
+const postHistoryHandler = async (token, dataDispatch, videoItem) => {
+  try {
+    const {
+      data: { history },
+    } = await postHistoryService(token, videoItem);
+    history &&
+      dataDispatch({
+        type: Abbreviations.ADD_TO_HISTORY,
+        payload: history,
+      });
+  } catch (error) {
+    console.log("Ching!", error);
```

Comment:
Use console.error to console all of your errors that way it would be easy for you to distinguish between other console.logs and errors